<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите один из вариантов:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Line">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Линейная функция</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Parabola">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Парабола</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Trigeometry">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Тригонометрические
функции</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Giperbola">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Гипербола</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
